azurelinuxagent/common/version.py

```diff
@@ -116,6 +117,28 @@ def get_distro():
     return osinfo
 
 
+def get_lis_version():
+    """
+    This uses the Linux kernel's 'modinfo' command to retrieve the "version"
+    field for the "hv_vmbus" kernel module (the LIS drivers). This is the
+    documented method to retrieve the LIS version.
+    """
+    lis_version = "Unknown"
+    try:
+        command_output = shellutil.run_command(["modinfo", "-F", "version", "hv_vmbus"], log_error=True)
```

Comment:
we need better logging for errors here. since exceptions are being ignored, should the command fail we would get a message saying that modinfo failed; the message should be more explicit, something saying that the version of the lis driver could not be retrieved.

Reply:
The logger module is not currently imported in `version.py`. However, as far as I can tell, `run_command` imports your logger module and with `log_error=True` records the logs. I am actually unsure what cautions this codebase currently takes with `run_command`. At least `get_systemd_version()` simply returns whatever `run_command` returns, with no wrapping try/catch, but `_collect_logs()` catches `Exception` and then includes it in event message (but does not log it with the logger module).

How would you like exceptions thrown by `run_command` handled here?

Reply:
@narrieta So this was the cause of those failures. Based on my understanding of the other tests, I've changed `log_error` to `False` (well, I removed the argument and am using its default now).